content/2-templating/2-styling/emberPolaris/css-style.gjs

```diff
@@ -0,0 +1,16 @@
+<template>
+  <h1 class="title">I am red</h1>
+  <button style="font-size: 10rem;">I am a button</button>
+</template>
+
+/**
```

Comment:
I think we're actually planning on putting `style` _in_ the `<template>

so for this demo, since it would work today with glimmer-scoped-css and (soon) with ember-scoped-css, the `<style>` should go in the `<template>`, and then we can ditch the inline `style` 🎉

Reply:
I think the example serves to showcase both inline `style` and `<style>`, as seen [here](https://github.com/spuxx1701/component-party.dev/blob/feat/ember-polaris/content/2-templating/2-styling/emberOctane/css-style.hbs#L2). So should we get rid of the content entirely and just do `<template><style></style></template>`? This *is* a preview, so I feel we can do that.

Reply:
So my suggestion would be

```ts
<template>
  <h1 class="title">I am red</h1>
  <button style="font-size: 10rem;">I am a button</button>
  
  <style>
    .title {
      color: red;
    }
  </style>
</template>
```